subprojects/wrapper/wrapper.gradle

```diff
@@ -26,18 +28,9 @@ dependencies {
     integTestRuntime rootProject.configurations.testRuntime.allDependencies
 }
 
-ext {
-    wrapperBuildReceipt = "build-receipt.properties"
-}
-
-task wrapperBuildReceiptResource() {
-    inputs.property 'version', { version }
-    outputs.file { "${generatedTestResourcesDir}/${wrapperBuildReceipt}" }
-    outputs.cacheIf { true }
-    doLast {
-        generatedTestResourcesDir.mkdirs()
-        new File(generatedTestResourcesDir, wrapperBuildReceipt).text = "versionNumber=${version}"
-    }
+task wrapperBuildReceiptResource(type: Copy) {
+    into generatedTestResourcesDir
+    from project(':versionInfo').createTestBuildReceipt.outputs.files
```

Comment:
Is there a better way to publish these outputs from the `:versionInfo` project? Not having to call `evaluationDependsOn()` would be a plus, too.